tools/bio3d/visualize_pc.xml

```diff
@@ -0,0 +1,141 @@
+<tool id="bio3d_pca_visualize" name="PCA visualization" version="@VERSION@">
+    <description>of principle components of motions</description>
```

Comment:
can/should be add "protein"/"structure" to the description?

Reply:
Thanks @bgruening I will add this